models/User.js

```diff
@@ -47,7 +47,17 @@ module.exports = (sequelize, DataTypes) => {
       allowNull: true,
     },
   };
-  const User = sequelize.define('User', userSchema);
+
+  const User = sequelize.define('User', userSchema, {
+    classMethods: {
+      associate(models) {
+        User.hasMany(models.Article, {
+          foreignKey: 'userId',
+          as: 'todoItems',
```

Comment:
Little mistake here, shouldn't be `todoItems`.

Reply:
This was a mistake. Thanks for spotting it out.